modules/sql/src/main/scala/gem/sql/FiniteDuration.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.sql
+
+/** Minimal time classes to support reading from the enum
+  * tables.
+  */
+sealed trait FiniteDuration extends Product with Serializable {
+  def toMillis: Long
+
+}
+
+object FiniteDuration {
+  final case class Seconds(toMillis: Long) extends FiniteDuration
+  final case class Milliseconds(toMillis: Long) extends FiniteDuration
+
+  def fromSeconds(bd: Long): FiniteDuration.Seconds =
+    Seconds(bd)
```

Comment:
Here I think we should switch to a `BigDecimal` and do the conversion in the trait as above but as is this would need to multiply `bd` by `1000` to get milliseconds.

```scala
object FiniteDuration {
  final case class Seconds(toBigDecimal: BigDecimal) extends FiniteDuration
  final case class Milliseconds(toBigDecimal: BigDecimal) extends FiniteDuration

  from Seconds(bd: BigDecimal): FiniteDuration.Seconds =
    Seconds(bd)

  from Milliseconds(bd: BigDecimal): FiniteDuration.Milliseconds =
    Microseconds(bd)
}
```